externals/kyuubi-jdbc-engine/src/main/scala/org/apache/kyuubi/engine/jdbc/operation/ExecuteStatement.scala

```diff
@@ -37,6 +39,8 @@ class ExecuteStatement(
   private val operationLog: OperationLog = OperationLog.createOperationLog(session, getHandle)
   override def getOperationLog: Option[OperationLog] = Option(operationLog)
 
+  @volatile private var jdbcStatement: Statement = _
```

Comment:
can it be a val？

Reply:
It seems not, we need to set value in executeStatement